waf_unsupported_target_test.go

```diff
@@ -3,15 +3,16 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-present Datadog, Inc.
 
-//go:build (!linux && !darwin) || (!amd64 && !arm64)
+//go:build ((!linux && !darwin) || (!amd64 && !arm64)) && !go1.22
```

Comment:
same as before but with the extra spicy sauce about us needing to uniformize formatter config